### DISCUSSION TITLE
Can you elaborate on the billing metrics?

### DISCUSSION THREAD

**Discussion by lilnasy** [rivet-dev/rivet#1684] [Q&A] [answered]
![IMG_0988](https://github.com/user-attachments/assets/7e1bc8a8-258c-4809-b502-10c6f38d561e)

The billing section mentions 200,000 included actors. I find it unclear what it means concretely. What's one actor? Is it an instantiation? How does the billing scale if all rpc calls for an entire app are sent to a single instance and it performs some routing logic to virtualize rooms? Are individual requests billed?

**Comment by NicholasKissel** [comment, marked as answer]
Requests are not billed, each actor within the project is billed. For example if you created a chat application each user is an actor, and the chat is an actor. So the usage of actors is based on the architecture of the project. Let me know if you have any more questions.

**Reply by lilnasy** [reply]
Thanks, that does help!

Can you elaborate on the use of the word "existing"?

Say I build a to-do app just for myself - 1 user actor, 1 app actor. Does the user actor go "out of existence" when I close the app? If I login the day after, does rivet consider it a completely different user actor, or can the older one be resumed?

I was hoping to find these answers here: https://rivet.gg/docs/lifecycle. However, that document goes directly into the APIs, and there's not much documentation on what the conceptual lifecycle of an actor is.

**Reply by NicholasKissel** [reply]
No the actor remains the same and goes to sleep. You are also not billed for runtime when the actor is sleeping.
